src/backend/utils/indexer.js

```diff
@@ -67,23 +67,16 @@ const search = async (textToSearch) => {
     body: { hits },
   } = await client.search({
     from: 0,
+    _source: ['null'],
```

Comment:
shouldn't this by `_source: ['id']`?

Reply:
Not really, _source should not include anything since we only saved 'text' key.
I tried to do like _source: [''] or [null] but it didn't work.
Any suggestion?

Reply:
it worked with ['id'] for me

Reply:
Yes, it works because there is no id~ so it doesn't mean anything. 
If you want, I can change it.